packages/perspective-viewer-d3fc/src/js/axis/mainAxis.js

```diff
@@ -13,41 +13,60 @@ import {extentLinear as customExtent} from "../d3fc/extent/extentLinear";
 export const scale = () => d3.scaleLinear();
 
 export const domain = () => {
-    let valueName = "mainValue";
+    let lowerValueName = "mainValue";

```

Comment:
I'm not keen on this, since it's very specific. Why not make it `valueNames` (an array), with a `valueName` property exposed as a shortcut for backwards compatibility? Then it just needs to calculate the min/max extent over all the valueNames.

Reply:
Switched to valueNames array as suggested